src/block-components/image/index.js

```diff
@@ -112,6 +115,10 @@ export const Image = props => {
 		defaultWidth={ props.defaultWidth }
 		defaultHeight={ props.defaultHeight }
 
+		figcaption={ attributes.figcaption }
+		figcaptionShow={ attributes.figcaptionShow }
+		figcaptionClasses={ props.figcaptionClasses }
```

Comment:
I think you mean `figcaptionClassnames`
```suggestion
		figcaptionClassnames={ props.figcaptionClasses }
```